pkg/version/version_parser_test.go

```diff
@@ -583,3 +583,164 @@ func TestLess(t *testing.T) {
 		})
 	}
 }
+
+func TestEqual(t *testing.T) {
```

Comment:
Instead of duplicating all the test cases from TestLess, maybe easier to just test both things in the same test? Maybe with a different sub-test?

I think maintaining the test cases is most of the work in here so we are better off duplicating them less.